files/templates/header.html

```diff
@@ -118,7 +118,7 @@
 
 								<a class="dropdown-item" rel="nofollow noopener noreferrer" href="https://github.com/themotte/rDrama"><i class="fab fa-github fa-fw mr-3"></i>Source Code</a>
 
-								<a class="dropdown-item" href="/support"><i class="fas fa-hippo fa-fw mr-3"></i>Support {{SITE_NAME}}</a>
```

Comment:
Oh man, I apologize for changing the requirements here, but now that I see it *should* have been SITE_NAME, y'know what, that's a better option. But then why isn't that working? On my local codebase it just shows up as blank.

![image](https://user-images.githubusercontent.com/160754/198155279-e322a72f-6acb-4b00-b6c2-78778bdcdc73.png)

Any way you can track down why SITE_NAME isn't working instead? It should say "Support The Motte", and I'd rather not hardcode that if possible, but this *should* have worked and isn't.

(I probably just did something dumb here honestly)

Reply:
looks like it should be SITE_TITLE not SITE_NAME - fixed